FHICORC.Backend/FHICORC.Application.Models/Enums/HashTypeEnum.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FHICORC.Core.Services.Enum
+{
+    public enum HashTypeEnum
+    {
+        NONE = 0,
```

Comment:
Use PascalCase for Enums

Reply:
The reason this is uppercase, is because it is uppercase on the EU DCCG.
![image](https://user-images.githubusercontent.com/44182405/172305335-3ca1488e-2bb6-419d-9582-135c417e8568.png)

I use this Enum to map over the the uppercase string from the DCCG to Enum.

Reply:
Will you not be able to use something like Enum.Parse with ignore case, or if it is during deserialization either the JsonProperty attribute or JsonSerializerOptions?